src/kirby/components/empty-state/empty-state.component.html

```diff
@@ -0,0 +1,8 @@
+<article>
+  <kirby-icon *ngIf="iconName" [name]="iconName" size="lg" outline="true"></kirby-icon>
+  <h3 *ngIf="title">{{ title }}</h3>
+  <p *ngIf="text">{{ text }}</p>
+  <div class="button-wrapper">
```

Comment:
Consider renaming (you don't know if the target project will embed a button or something else):
```suggestion
  <div class="content">
```

Reply:
Fixed